cli/compose/schema/data/config_schema_v3.8.json

```diff
@@ -538,6 +538,7 @@
           }
         },
         "labels": {"$ref": "#/definitions/list_or_dict"},
+        "driver": {"type": "string"},
```

Comment:
could you squash this commit together with the one that re-generates the bindata?

Reply:
I've now split this PR in 3 commits:
1. Modifying the schema itself and updating bindata
2. Making use of the changes to the schema in the relevant Go code
3. Adding a test case